spirv_msl.cpp

```diff
@@ -27,13 +27,20 @@ using namespace std;
 
 static const uint32_t k_unknown_location = ~0;
 
-CompilerMSL::CompilerMSL(vector<uint32_t> spirv_)
+CompilerMSL::CompilerMSL(vector<uint32_t> spirv_, vector<MSLVertexAttr> *p_vtx_attrs,
+                         std::vector<MSLResourceBinding> *p_res_bindings)
```

Comment:
Inconsistent std:: usage.
Also, must the arguments be non-const?
It seems like CompilerMSL ends up storing pointers owned by these arguments. Is this behavior documented?